app/comment/serializers.py

```diff
@@ -0,0 +1,9 @@
+
+from rest_framework import serializers
+from .models import Comment
+
+
+class CommentSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Comment
+        fields = '__all__'
```

Comment:
`fields = '__all__'` is not recommended. It can lead to unintended side effects when the `model` has additional fields added to it.